Backtrace.podspec

```diff
@@ -31,8 +31,6 @@ Pod::Spec.new do |s|
   s.osx.public_header_files = ["Backtrace-macOS/**/*.h*"]
   s.tvos.public_header_files = ["Backtrace-tvOS/**/*.h*"]
 
-  s.ios.dependency "Cassette", '1.0.0-beta5'
```

Comment:
Check if there's any superfluous `#if os(iOS) || os(OSX)` left in prod/test code. I know they were a bunch of 'm

Reply:
I'll take a look :)